package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "@rbxts/datastore2",
-	"version": "1.3.3",
+	"version": "1.4.0",
```

```suggestion
	"version": "1.4.0-ts.0",
```

I've started versioning Lua packages with .d.ts types as: `{LUA_VERSION}-ts.{TS_PATCH}`.

---

This way, if we need to make an update to the types, it can be `1.4.0-ts.1` and still clear that it's based on datastore2 `1.4.0`